std/algebra/emulated/fields_bn254/e12_pairing.go

```diff
@@ -0,0 +1,136 @@
+package fields_bn254
+
+import (
+	"github.com/consensys/gnark-crypto/ecc/bn254"
+	"github.com/consensys/gnark/std/math/emulated"
+)
+
+type GTEl = E12
```

Comment:
I would move this type alias and `NewGTEl` to `sw_bn254` package. Then we have all related types (or aliases) in a single package and it is easier to comprehend.

Reply:
I would rather keep field aliases separate from curve aliases. `fields_bn254` package could be used independently of `sw_bn254` package.

Reply:
Definitely, we can use it separately. What I was proposing is to keep `E12` type in this package as is. All the methods on `E12` will stay here as is. We will only move the alias `type GTEl = E12` into `sw_bn254` package (as `type GTEl = fields_bn254.E12`. Actually it would be similar how it currently is for the 2-chains (see https://github.com/ConsenSys/gnark/blob/f2b7910c6a50c6272806bf21576e61efb2667167/std/algebra/native/sw_bls12377/pairing.go#L28)

Reply:
I see! ok that makes sense